recipes/jep/fix-split.patch

```diff
@@ -13,8 +13,6 @@ index ccbda7e..f6b600f 100644
  
                  result = shell('sw_vers -productVersion')
 -                if target.split('.')[:2] != result.stdout.split('.')[:2]:
-+                print("type target: {}".format(type(target)))
-+                print("type result.stdout: {}".format(type(result.stdout)))
```

Comment:
This may mess up the patch since the number of lines changed. I am not 100% sure.

Reply:
Looks like it did. Glad you fixed it!